stock_putaway_storage_type_strategy/models/stock_location.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, models, fields
+from odoo.osv.expression import AND, OR
+
+
+class StockLocation(models.Model):
+
+    _inherit = 'stock.location'
+
+    pack_storage_strategy = fields.Selection(
+        selection=[('none', 'None'),
+                   ('ordered_locations', 'Ordered Children Locations')],
```

Comment:
I cannot find any business use case where we would want an ordered_locations strategy

Reply:
Take the first bin in the row in front of you everytime (put in the bin closest to you). As you have constraints, it will take the next available in the row.

Reply:
I just cannot figure out who would like to work like that ;-) Not an issue of course. Just a side note.